packages/analytics-types/src/config/browser.ts

```diff
@@ -26,34 +73,97 @@ interface InternalBrowserConfig {
   lastEventTime?: number;
   lastEventId?: number;
   transportProvider: Transport;
-  pageCounter?: number;
```

Comment:
Is there any cases where customers need to set it at `init()`? If no, we should keep it as internal.

Reply:
Nope. However, this is bundled with page view plugin. We set the page counter in the page view plugin, we cannot access pageCounter if it's internal.